library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -138,6 +138,11 @@ public void onInfoWindowClick(Marker marker) {
     callback.onMapBoundsReady(builder.build());
   }
 
+  @Override
```

Comment:
minor: can you put this in the same line as the method signature to match the code formatting, please?